Assets/AnkrSDK/Scripts/Mobile/MobileWalletHandler.cs

```diff
@@ -21,7 +22,7 @@ public UniTask<WalletsStatus> GetWalletsStatus()
 			var isConnected = _walletConnect?.Session?.Accounts?.Length > 0;
 			var status = new WalletsStatus
 			{
-				{"Metamask", isConnected}
+				{Wallet.Metamask, isConnected}
```

Comment:
why there is only metamask here?

Reply:
It's a stub just for now. I don't know how to track sessions on different wallets on mobile devices. And in current version of wallet connect we can't specify what wallet should be opened by deeplink. It means that this feature will not work on mobile devices.

Reply:
We cant specify wallet because it is not needed on Android and not supported on IOS. 
Android has system built-in popup which will suggest user to select wallet. And on IOS user selects wallet from the list provided by developer.
I suggest we add Wallet.Unknown for example, so we don't confuse ourselves or our users

Reply:
We have Wallet.None. Can use that.

Reply:
It is not the same as Unknown. We use to None as a placeholder if user/dev has not selected a specific wallet. 
Unknown is for the case then we don't know what wallet is being used